plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py

```diff
@@ -3,36 +3,43 @@
 import sys
 import tempfile
 from pathlib import Path
-from typing import List
+from typing import Callable, List, Optional
 
-import pytest
 from hydra.core.plugins import Plugins
 from hydra.plugins.launcher import Launcher
 from hydra.test_utils.launcher_common_tests import (
+    IntegrationTestSuite,
     LauncherTestSuite,
 )
 from hydra.test_utils.test_utils import chdir_plugin_root
-from omegaconf import DictConfig, OmegaConf
+from omegaconf import OmegaConf
 
+import pytest
 from hydra_plugins.hydra_ray_launcher.ray_aws_launcher import (
-    RayAWSLauncher,  # type: ignore
+    RayAWSLauncher,  # type: ignore,
 )
 from plugins.hydra_ray_launcher.hydra_plugins.hydra_ray_launcher.ray_launcher_util import (
-    ray_down,
+    ray_rsync_down,
     ray_tmp_dir,
     ray_up,
+    ray_working_dir, ray_down,
 )
-from plugins.hydra_ray_launcher.tests.ray_test_utils import ray_integration_test
 
 chdir_plugin_root()
 
 win_msg = "Ray doesn't support Windows."
 
 cluster_name = "IntegrationTest"
 
+temp_yaml = ""
+temp_remote_dir = ""
+remote_working_dir = ""
+
+sweep_dir = "pytest_dir"
+
 default_test_override = [
     f"hydra.launcher.params.ray_cluster_cfg.cluster_name={cluster_name}",
-    "hydra.launcher.params.code_dir=''",
+    "hydra.launcher.params.code_sync_cfg.code_dir=''",
```

Comment:
per my previous comment, this can become:
```suggestion
    "hydra.launcher.params.code_sync_cfg.code_dir=null",
```
But also, since empty/null is the default, you don't even need this.